src/CONST.js

```diff
@@ -471,7 +471,6 @@ const CONST = {
         MAX_PENDING_TIME_MS: 10 * 1000,
     },
     NVP: {
-        IS_FIRST_TIME_NEW_EXPENSIFY_USER: 'isFirstTimeNewExpensifyUser',
         BLOCKED_FROM_CONCIERGE: 'private_blockedFromConcierge',
```

Comment:
Were/are these set with something other than `NameValuePair.set`?

Reply:
If you are asking about `BLOCKED_FROM_CONCIERGE` it's a private NVP that is read-only from the front end.

Reply:
Oh! Nice, I will get rid of it then. Nice catch!